home/components/Profile.js

```diff
@@ -324,7 +325,8 @@ export default class Profile extends React.Component {
           media = await ImagePicker.launchImageLibraryAsync(mediaOptions);
         }
         if (!media.cancelled) {
-          Store.dispatch(ProfileActions.setImage(media.uri));
+          const asset = await MediaLibrary.createAssetAsync(media.uri);
```

Comment:
This copies the photo to the camera roll, right? We don't want to duplicate photos selected from the camera roll to begin with -- we just need to save a copy of the photo elsewhere on disk so that it doesn't disappear.

Reply:
that has _really_ unpredictable behavior across platforms. Im pretty sure it doesn't work on iOS Camera the same as it does on iOS Photo Gallery media.

Reply:
Copying a file should have the same behavior everywhere, what specifically is unpredictable?